Gruntfile.coffee

```diff
@@ -87,13 +87,14 @@ module.exports = (grunt) ->
 
 	)
 
-	@registerTask(
-		"init"
-		"Only needed when the repo is first cloned"
-		[
-			"modernizr"
-		]
-	)
+# Remove modernizr temp
+#@registerTask(
+#		"init"
+#		"Only needed when the repo is first cloned"
+#		[
+#			"modernizr"
```

Comment:
Not sure if some themes are calling this directly anymore, but it might be better to just remove this entry from the task rather then removing it till it is fixed